### ISSUE TITLE
MissingFileError for `location_field/Leaflet.GoogleMutant.js.map`

### ISSUE THREAD

**Issue by jvacek** [caioariede/django-location-field#168]
I think this might be my own configuration issue actually. I have installed the app into the cookiecutter django project with whitenoise enabled, and it's failing on this file when trying to build my production docker environment (local collectstatic works fine)

```
Post-processing 'location_field/Leaflet.GoogleMutant.js' failed!

Traceback (most recent call last):
  File "/app/manage.py", line 31, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 446, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.11/site-packages/django/core/management/__init__.py", line 440, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 402, in run_from_argv
    self.execute(*args, **cmd_options)
  File "/usr/local/lib/python3.11/site-packages/django/core/management/base.py", line 448, in execute
    output = self.handle(*args, **options)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 209, in handle
    collected = self.collect()
                ^^^^^^^^^^^^^^
  File "/usr/local/lib/python3.11/site-packages/django/contrib/staticfiles/management/commands/collectstatic.py", line 154, in collect
    raise processed
whitenoise.storage.MissingFileError: The file 'location_field/Leaflet.GoogleMutant.js.map' could not be found with <whitenoise.storage.CompressedManifestStaticFilesStorage object at 0x7f7c720ba5d0>.

The JS file 'location_field/Leaflet.GoogleMutant.js' references a file which could not be found:
  location_field/Leaflet.GoogleMutant.js.map

Please check the URL references in this JS file, particularly any
relative paths which might be pointing to the wrong location.
```

I can see the file is present where it should be

```
# docker-compose -f production.yml run --rm django ls "/usr/local/lib/python3.11/site-packages/location_field/static/location_field"
Starting flamerelay_postgres_1 ... done
Starting flamerelay_redis_1    ... done
PostgreSQL is available
@googlemaps  Leaflet.GoogleMutant.js  js  leaflet  leaflet-geosearch  leaflet-src.js.map
```

**Comment by jvacek** [comment]
The only reference to the file I can find is here https://github.com/caioariede/django-location-field/blob/4f12f424a0e57b0d4e5d37824fa3f98c12fd485c/location_field/static/location_field/js/form.js#L270

**Comment by leikoilja** [comment]
Started to see this issue as well now. 
Suspect that it was introduced in https://github.com/caioariede/django-location-field/pull/157/files but i didn't dive into the implementation.
Rollback to `django-location-field==2.1.1` fixed it for me, for the time being

**Comment by caioariede** [comment]
This has been fixed in version 2.7.1 :)
